Tasks/AzureRmWebAppDeployment/AzureUtility.ps1

```diff
@@ -58,7 +58,7 @@ function Get-AzureRMWebAppConnectionDetailsWithSpecificSlot
           [String][Parameter(Mandatory=$true)] $resourceGroupName,
           [String][Parameter(Mandatory=$true)] $slotName)
 
-    Write-Host (Get-LocalizedString -Key "Getting connection details for azureRM WebApp:'{0}' under Slot:'{1}'." -ArgumentList $webAppName, $slotName)
+    Write-Host (Get-VstsLocString -Key "GettingconnectiondetailsforazureRMWebApp0underSlot1" -ArgumentList $webAppName, $slotName)
 
     $kuduHostName = $webAppName + "-" + $slotName + ".scm.azurewebsites.net"
     Write-Verbose "`t Using KuduHostName:'$kuduHostName' for azureRM WebApp:'$webAppName' under Slot:'$slotName'."
```

Comment:
fyi - this type of tracing can be simplified. Adding a call to the function `Trace-VstsEnteringInvocation $MyInvocation` will walk the psbound parameters hashtable and give you tracing.